erts/emulator/beam/ops.tab

```diff
@@ -1425,7 +1425,11 @@ sorted_put_map_exact F Src=s Dst Live Size Rest=* => \
 sorted_put_map_exact F Src Dst Live Size Rest=* => \
 	      move Src x | update_map_exact F x Dst Live Size Rest
 
+new_map Dst Live Size Rest=* | are_map_keys_literals(Size, Rest) => \
+   gen_new_map_lit(Dst, Live, Size, Rest)
+
 new_map d I I
+i_new_map_lit d I q
```

Comment:
I wasn't sure about where this should be placed. It looks like the operations are grouped by release they came to be. On the other hand, it's nice to have all the different map operations together.